framework/auth/views.py

```diff
@@ -150,7 +154,8 @@ def auth_login(auth, **kwargs):
 
     status_message = request.args.get('status', '')
     if status_message == 'expired':
-        status.push_status_message('The private link you used is expired.')
+        status.push_status_message('The private link you used is expired.  Please <a href="/settings/account/"> +'
```

Comment:
This would take a bit of effort to use a direct link.  Seems like a separate PR would be a good idea.